sweepbatcher/presigned.go

```diff
@@ -0,0 +1,602 @@
+package sweepbatcher
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
+)
+
+// ensurePresigned checks that we can sign a transaction sweeping the inputs of
+// this group only. This method is called when a group of sweeps of new swap is
+// being added to the batch. If the batch can be either empty or existing. This
+// is know to the caller and affects allowNonEmptyBatch flag.
```

Comment:
nit: known

Reply:
This part of the godoc was removed in the updated version.